src/foundation.cs

```diff
@@ -12226,13 +13150,22 @@ interface NSUnitTemperature : NSSecureCoding {
 		NSDimension BaseUnit { get; }
 	}
 
-#if MONOMAC
+#if !WATCH && !TVOS
+	delegate void FileProviderMessageInterfacesCompletionHandler (NSFileProviderMessageInterface [] interfaces, NSError error);
+
+	[Mac (10,8), iOS (11,0), NoWatch, NoTV]
 	partial interface NSFileManager {
 
 		[MountainLion, Export ("trashItemAtURL:resultingItemURL:error:")]
 		bool TrashItem (NSUrl url, out NSUrl resultingItemUrl, out NSError error);
+
+		[NoWatch, NoTV, Mac (10,13), iOS (11,0)]
+		[Async, Export ("getFileProviderMessageInterfacesForItemAtURL:completionHandler:")]
+		void GetFileProviderMessageInterfaces (NSUrl url, FileProviderMessageInterfacesCompletionHandler completionHandler);
```

Comment:
Gotcha.  It had been pointed out one one of the other (longer) handlers that we needed the delegate because of the multiple parameters and I made all of 2 parameters (with one as an NSError) into delegates, so there's probably others I can change back as well.

I'll go through ll the delegates and convert any that match back to Action<T, V>